### ISSUE TITLE
Inequality functionality for `norm_num`

### ISSUE THREAD

**Issue by thorimur** [leanprover-community/mathlib4#1567]
This is an issue for tracking inequality functionality needed in `norm_num`, including `≤`, `<`, and `≠`.

For `≠`, the following example should work:
```
example {α} [DivisionRing α] [CharZero α] : (-1:α) ≠ 2 := by norm_num
```

**Comment by hrmacbeth** [comment]
Thanks for opening this issue @thorimur, indeed this is very important functionality.  I removed the Github link between this issue and #1568 because if they're linked then Github will "close" this issue when #1568 is merged, whereas we want it to stay open until the `≤` and `<` functionality is also implemented.

**Comment by digama0** [comment]
Fixed in #1568
